doc/nrf/libraries/bluetooth_services/mesh/gen_onoff_cli.rst

```diff
@@ -10,17 +10,64 @@ Generic OnOff Client
 The Generic OnOff Client model remotely controls the state of a :ref:`bt_mesh_onoff_srv_readme` model.
 
 Extended models
-===============
+***************
 
 None.
 
 Persistent storage
-==================
+******************
 
 None.
 
+Shell commands
+**************
+
+The Bluetooth mesh shell subsystem provides set of commands to interract with the Generic OnOff Client model instantiated on a device.
+
+To make these commands available, enable :kconfig:`CONFIG_BT_MESH_SHELL` and :kconfig:`CONFIG_BT_MESH_SHELL_ONOFF_CLI`.
+
+``mdl_onoff instance get-all``
+------------------------------
+
+	Print all instances of the Generic OnOff Client model on the device.
+
+
+``mdl_onoff instance set <elem_idx>``
+-------------------------------------
+
+	Select the Generic OnOff Client model instantiated on the specified element id. This instance will be used in message sending. If no model instance was selected, the first model instance found on the device will be used by default.
+
+	* ``elem_idx``: Element index where to find the model instance.
+
+
+``mdl_onoff get``
+-----------------
+
+	Get the OnOff state value on the bound server.
+
+
+``mdl_onoff set <onoff> [transition_time_ms [delay_ms]]``
+----------------------------------------------------------
+
+	Set the OnOff state value in the server and wait for a response.
+
+	* ``onoff``: OnOff state value to set. Use `on`, `enable`, or any non-zero value to enable the state.
```

Comment:
```suggestion
	* ``onoff``: OnOff state value to set. Use *on*, *enable*, or any non-zero value to enable the state.
```
Use * instead of ` for italics: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/contribute/documentation/index.html#content-highlighting. Also in line 64.